src/hotspot/cpu/riscv/interp_masm_riscv.cpp

```diff
@@ -441,7 +441,14 @@ void InterpreterMacroAssembler::dispatch_base(TosState state,
                                               Register Rs) {
   // Pay attention to the argument Rs, which is acquiesce in t0.
   if (VerifyActivationFrameSize) {
-    Unimplemented();
+    Label L;
+    sub(t1, fp, esp);
+    sub(t1, t1, frame::metadata_words * wordSize); // Exclude 2 frame metadata words
+    int min_frame_size = (frame::link_offset - frame::interpreter_frame_initial_sp_offset) * wordSize;
```

Comment:
Could we also remove the above `sub`, and add `frame::metadata_words * wordSize` in this statement? it's not necessary to calculate the value in the generated code and seems to me the code will be more straight.

Reply:
Yeah, I agree that will be cleaner.
```
int min_frame_size = (frame::link_offset - frame::interpreter_frame_initial_sp_offset + frame::metadata_words) * wordSize;
```

Reply:
Thanks, the second `sub` calculation has been merged to `min_frame_size ` statement.